.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: Bazel build and test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: gcr.io/gapic-images/googleapis-bazel
```

Comment:
Please consider making this versioned and also somehow corelated with the cache. Note every time an image is updated the cache must be completely invalidated. I.e. cache and image version should go together.

Reply:
Done!